src/tabset/tabset.spec.ts

```diff
@@ -165,7 +168,7 @@ describe('ngb-tabset', () => {
     const tabTitles = getTabTitles(fixture.nativeElement);
 
     expect(tabTitles[0].textContent).toMatch(/foo/);
-    expect(tabTitles[1].innerHTML).toMatch(/<b>bar<\/b>/);
+    expect(tabTitles[1].innerHTML).toMatch(/<b( class=""|)>bar<\/b>/);
```

Comment:
Any insights on why this tests needs changing? I'm not sure why, all of the sudden, this element is being touched... We even don't have animations on titles :-/

Reply:
I believe its the angular animations module adding pseudo classes to elements, as on browsers in saucelabs that don't support have the web animations library the class wasn't added.